crates/swc_ecma_parser/tests/tsc/awaitCallExpression9_es5.ts

```diff
@@ -0,0 +1,16 @@
+// @target: ES5
```

Comment:
Why did you add this file? Please remove

Reply:
To explicitly test the behaviour for arrays, however the current solution doesn't really require it